6_nodejs/week1/avg.js

```diff
@@ -0,0 +1,55 @@
+//  Parse Arguments using process.argv
+let avg;
+function average() {
+  let sum = 0;
+  let count = 0;      
+  if(process.argv.length < 5){
+        console.log("please enter at least three numbers");
+        return
+  }
+  for (let j = 2 ; j < process.argv.length; j++) {
+    const num = parseInt(process.argv[j]);
+    if(isNaN(num)){
+        console.log(`${process.argv[j]} is not a number`);
+        return
+    };
+    sum += num;
+    count++
+  };
+  avg = sum / count;
+  console.log(`avg: ${avg}`);
+}
+average()
+
+
+//  Passing Arguments Using Yargs
+import yargs from 'yargs'
+import { hideBin } from 'yargs/helpers'
+const argv = yargs(hideBin(process.argv))
```

Comment:
Good job 👍  If you want to make the yargs implementation more flexible, you can also allow users to input an array of numbers, making it able to handle more than 3 numbers